controller/server_controller.py

```diff
@@ -125,31 +123,45 @@ def cull_idle_jupyter_servers(body, name, namespace, logger, **kwargs):
     js_server_status = get_js_server_status(body)
     custom_resource_api = get_api(config.api_version, config.custom_resource_name)
     idle_seconds = int(body["status"].get("idleSeconds", 0))
+    now = pytz.UTC.localize(datetime.utcnow())
+    last_activity = now if js_server_status is None else js_server_status.get("last_activity", now)
+    try:
+        jupyter_server_started = datetime.fromisoformat(
+            body["metadata"]["creationTimestamp"].replace("Z", "+00:00")
+        )
+    except KeyError:
```

Comment:
The reason I did this is because the culling timer runs continuously every X seconds. I think it is possible (during startup and/or shutdown) for this field to be empty (even only momentarily). But this may be enough time for the culling to run and trip on this exception.

Reply:
I will take this out and put it in if we see errors later.